docs/reference/docs/delete-by-query.asciidoc

```diff
@@ -54,6 +54,10 @@ conflict if the document changes between the time when the snapshot was taken
 and when the delete request is processed. When the versions match the document
 is deleted.
 
+NOTE: Since `internal`versioning does not support the value 0 as a valid
+version number, documents with version equal to zero cannot be updated using
+`_update_by_query` and will fail the request.
```

Comment:
That was intended to check if the reviewer did a good job. Or it might just be a copy/paste error... :)